### DISCUSSION TITLE
Using ag-grid-vue3 with Quasar

### DISCUSSION THREAD

**Discussion by kannaiah** [quasarframework/quasar#13568] [General - Components / Directives / etc] [answered]
Hi,
I'm trying to use ag-grid-vue3 with Qusar.
Although I do not see any errors nor warnings table is not being rendered.

Example usage:
https://codesandbox.io/s/green-river-ou0ytt?file=/src/pages/Index.vue

**Comment by suleimanas** [comment, marked as answer]
It's working fine, set the height of ag-grid-vue component to 100 you'll see it, already rendered

**Reply by kannaiah** [reply]
Looks like I cannot set height in `%` using `px`  showing the table.
I see now that there is a warning about this on ag-grid documentation.
